docs/layer_configuration.md

```diff
@@ -0,0 +1,342 @@
+    const char* profile_name_data = "VP_LUNARG_desktop_baseline_2023";
+    const VkBool32 emulate_portability_data = VK_TRUE;
+    const char* simulate_capabilities[] = {"SIMULATE_FEATURES_BIT", "SIMULATE_PROPERTIES_BIT", "SIMULATE_EXTENSIONS_BIT"};
+
+    const VkLayerSettingEXT settings[] = {
+        {kLayerName, kLayerSettingsProfileFile, VK_LAYER_SETTING_TYPE_STRING_EXT, 1, &profile_file_data},
+        {kLayerName, kLayerSettingsProfileName, VK_LAYER_SETTING_TYPE_STRING_EXT, 1, &profile_name_data},
+        {kLayerName, kLayerSettingsEmulatePortability, VK_LAYER_SETTING_TYPE_BOOL32_EXT, 1, &emulate_portability_data},
+        {kLayerName, kLayerSettingsSimulateCapabilities, VK_LAYER_SETTING_TYPE_STRING_EXT, static_cast<uint32_t>(std::size(simulate_capabilities)), simulate_capabilities}};
+
+    const VkLayerSettingsCreateInfoEXT layer_settings_create_info = {
+        VK_STRUCTURE_TYPE_LAYER_SETTINGS_CREATE_INFO_EXT, nullptr,
+        static_cast<uint32_t>(std::size(settings)), settings};
+
+    const VkApplicationInfo app_info = {
+        VK_STRUCTURE_TYPE_APPLICATION_INFO, nullptr,
+        "profiles_tests", VK_MAKE_API_VERSION(0, 1, 0, 0),
+        "profiles_tests", VK_MAKE_API_VERSION(0, 1, 0, 0),
+        VK_API_VERSION_1_3
+    };
+
+    const char* layers[] = {
+        "VK_LAYER_KHRONOS_validation", "VK_LAYER_KHRONOS_profiles"
+    };
+
+    const VkInstanceCreateInfo inst_create_info = {
+        VK_STRUCTURE_TYPE_INSTANCE_CREATE_INFO, &layer_settings_create_info, 0,
+        &app_info,
+        static_cast<uint32_t>(std::size(layers)), layers,
+        0, nullptr
+    };
+
+    VkInstance instance = VK_NULL_HANDLE;
+    VkResult result = vkCreateInstance(&inst_create_info, nullptr, &instance);
+```
+
+## Configuring Vulkan Layers using *Vulkan Configurator*
+
+Developers can configure layers through a graphical user interface. *Vulkan Configurator* allows full user control of Vulkan layers, including enabling or disabling specific layers, controlling layer order, changing layer settings, etc.
+*Vulkan Configurator* configures the layers by applying a global system configuration of the Vulkan loader and creating a vk_layer_settings.txt file that will be find by any layer.
+
+*Vulkan Configurator* can be used using command line to setup the system environment. Use the commannd `vkconfig --help` for more information in this case.
+
+We recommand the using *Vulkan Configurator* GUI approach for Vulkan application developers, it's the most effectively approach to switch between multiple layers configurations and quickly iterate during the development process.
+Additionally, *Vulkan Configurator* presents to the Vulkan application developers the layers found on the system and the settings of each layers, allowing Vulkan application developers to discover new fonctionnalities from the GUI without having to dig into each layer documentation.
```

Comment:
of each layers -> of each layer
fonctionnalities -> functionality

Reply:
Fixed